resources/enclosure.rb

```diff
@@ -32,3 +34,20 @@
 action :remove do
   remove
 end
+
+action :reconfigure do
+  item = load_resource
+  item.retrieve!
+  item.configuration
+end
+
+action :refresh do
```

Comment:
For these actions that don't test anything before taking an action, it might be a little misleading to people running the client with the --whyrun option. We say it's supported for the whole resource [here](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/oneview_resource_base.rb#L114), but these 2 new actions don't actually support it.

Is there anything we can add here as a test? Ie only do a reconfigure if it's in an inconsistent state or something?